code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -333,12 +333,6 @@ GLOBAL_LIST_INIT(runed_metal_recipes, list ( \
 		return
 	return ..()
 
-/obj/item/stack/sheet/runed_metal/attack(atom/target, mob/living/user)
```

Comment:
Stop please, this doesn't relate to bananium and is a silent revert not even mentioned on the commit, only slightly on the title with no supplied change log.

Reply:
it's literally in the commit and the PR desc